CHANGES.md

```diff
@@ -70,6 +70,8 @@ To be released.
  -  The difficulty was changed from representing the number of leading zeros of
     target number to representing a divisor to obtain the target number.
     [[#213]]
+ -  Removed unnecessary reader lock on `BlockChain<T>.StageTrasactions()`. [[#217]]
+ -  Improve concurrency of `BlockChain<T>.GetState()`. [[#217]]
```

Comment:
It should be `GetStates()`.

Reply:
Hmm… the change was actually made in `Append()`.  Would this explanation be just fine?

Reply:
It's true that it has been competing mostly with `GetStates`(), but it's probably better to write that the `Append()` because the change was made in `Append()` like you say.